others/postfix_evaluation.cpp

```diff
@@ -0,0 +1,159 @@
+/**
+ * @file
+ * @brief Evaluation of Postfix Expression
+ * @author [Darshana Sarma](https://github.com/Darshana-Sarma)
+ * @details
+ * Create a stack to store operands (or values).
+ * Scan the given expression and do following for every scanned element.
+ * If the element is a number, push it into the stack
+ * If the element is a operator, pop operands for the operator from stack.   
+ * Evaluate the operator and push the result back to the stack
+ * When the expression is ended, the number in the stack is the final answer
+ */
+#include <array>           // for std::array
+#include <iostream>        // for io operations 
+#include <cassert>  // for assert
+
+// Maximum size for stack
+const int MAX_SIZE = 20;
+
+// Defining stack
+std::array<int, MAX_SIZE> stack;
+
+// Index of top value in stack array
+int top;
+
+/**
+ * Pushing operand, also called the integer in the array to the stack
+ * @param operand integer value from the input array or evaluation
+ * @returns none
+ */
+void push(int operand){
+    top++;
+    stack[top]=operand;
+}
+
+/**
+ * Popping operand, also called the integer from the stack
+ * @param none
+ * @returns operand integer on top of stack
+ */
+int pop(){
```

Comment:
Same suggestion as above, maybe use

```suggestion
template <class T>
T pop() {
    T operand = stack[top]
```
However looking at your definition of stack, it seems that it might not be possible. In the case that using templates is not feasible, then ignore these comments.